osu.Game/Overlays/BeatmapSet/Header.cs

```diff
@@ -134,7 +134,7 @@ public Header()
                                                 {
                                                     title = new OsuSpriteText
                                                     {
-                                                        Font = OsuFont.GetFont(size: 37, weight: FontWeight.SemiBold, italics: true)
+                                                        Font = OsuFont.GetFont(size: 37.5f, weight: FontWeight.SemiBold, italics: true)
```

Comment:
If the artist text uses osu-web font size, I think this should too

Reply:
Good point, I got confused and looked at web's `line-height` instead of `font-size`. Will fix up in a sec.